core/misc/dialog.theme.css

```diff
@@ -2,16 +2,6 @@
  * Presentational styles for Backdrop dialogs.
  */
 
-/* Lock scrolling when an auto-resize dialog is open. */
-html.dialog-auto-resize {
```

Comment:
Why delete this? If deleting is required though, this class is only added in `dialog.js`, so should we remove that code too? 

As far as I can see, the only effect of this CSS is that without it, if you try to scroll while the dialog is resizing, the dialog achieves its final size off center, then animates to center, whereas if scrolling is locked it achieves its final size dead center. You have to try hard to get this effect, so we can live without it if this CSS causes other problems.

Reply:
> Why delete this?

Because this is the change that causes the jumping to top of page. So reverting this is essential.

> the only effect of this CSS is that without it, if you try to scroll while the dialog is resizing,

That's not correct. This "lock scrolling" causes the regression. I wouldn't remove the class, as it potentially is in use in some custom or contrib styles. And the class per se does no harm.

Reply:
In that case this is ready then.